src/code/z_play.c

```diff
@@ -354,13 +353,13 @@ void Play_Init(GameState* thisx) {
         !IS_CUTSCENE_LAYER) {
         if (CHECK_QUEST_ITEM(QUEST_KOKIRI_EMERALD) && CHECK_QUEST_ITEM(QUEST_GORON_RUBY) &&
             CHECK_QUEST_ITEM(QUEST_ZORA_SAPPHIRE)) {
-            gSaveContext.sceneLayer = 1;
+            gSaveContext.sceneLayer = SCENE_LAYER_CHILD_NIGHT;
         } else {
-            gSaveContext.sceneLayer = 0;
+            gSaveContext.sceneLayer = SCENE_LAYER_CHILD_DAY;
```

Comment:
```suggestion
            gSaveContext.sceneLayer = 0;
```

Because this one is conditional on not having all the spiritual stones, I don't think it's appropriate to call it `SCENE_LAYER_CHILD_DAY`, and should be given a special name when we name layers in the future.